### PR TITLE
Remove `Model` and rename `ModelExperimental -> Model`

### PR DESCRIPTION
**Pull request by ptrgags**
This PR does the following:

* Removes the old `Model.js` and some classes that were no longer used
* renames `ModelExperimental -> Model`
* Some tests were commented out since they use old glTF 1.0 assets. we'll have to try these assets with https://github.com/CesiumGS/gltf-pipeline/pull/619 and/or hand-edit them to remove the techniques.

There are still several other helper classes that need to be renamed/removed, but that will be the focus of a future PR to keep the reviews manageable.

@j9liu could you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Documentation/CustomShaderGuide/README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ const customShader = new Cesium.CustomShader({
 
 ## Applying A Custom Shader
 
-Custom shaders can be applied to either 3D Tiles or `ModelExperimental` as
+Custom shaders can be applied to either 3D Tiles or Model` as
```
Comment (j9liu) [inline Documentation/CustomShaderGuide/README.md:63]:
```suggestion
Custom shaders can be applied to either 3D Tiles or a `Model` as
```

File: Documentation/CustomShaderGuide/README.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ const tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
 }));
 
 // Applying to a model directly
-const model = Cesium.ModelExperimental.fromGltf({,
+const model = CesiumModel.fromGltf({,
```
Comment (j9liu) [inline Documentation/CustomShaderGuide/README.md:79]:
```suggestion
const model = Cesium.Model.fromGltf({,
```

File: Documentation/CustomShaderGuide/README.md [hunk 3]
```diff
@@ -707,7 +707,7 @@ property IDs, the behavior is undefined. For example:
 - Two properties with names `temperature ℃` and `temperature ℉` would both
   map to `metadata.temperature`, so the behavior is undefined
 
-When using the Point Cloud (`.pnts`) format in `ModelExperimental`, per-point
+When using the Point Cloud (`.pnts`) format in Model`, per-point
 properties are transcoded as property attributes. These property IDs follow
 the same convention.
```
Comment (j9liu) [inline Documentation/CustomShaderGuide/README.md:712]:
```suggestion
When using the Point Cloud (`.pnts`) format, per-point properties are
transcoded as property attributes. These property IDs follow the same convention.
```

File: Source/Scene/Cesium3DTileset.js [hunk 4]
```diff
@@ -111,7 +110,7 @@ import TileOrientedBoundingBox from "./TileOrientedBoundingBox.js";
  * @param {String} [options.debugHeatmapTilePropertyName] The tile variable to colorize as a heatmap. All rendered tiles will be colorized relative to each other's specified variable value.
  * @param {Boolean} [options.debugFreezeFrame=false] For debugging only. Determines if only the tiles from last frame should be used for rendering.
  * @param {Boolean} [options.debugColorizeTiles=false] For debugging only. When true, assigns a random color to each tile.
- * @param {Boolean} [options.enableDebugWireframe] For debugging only. This must be true for debugWireframe to work for ModelExperimental in WebGL1. This cannot be set after the tileset has loaded.
+ * @param {Boolean} [options.enableDebugWireframe] For debugging only. This must be true for debugWireframe to work for Model in WebGL1. This cannot be set after the tileset has loaded.
```
Comment (j9liu) [inline Source/Scene/Cesium3DTileset.js:113]:
```suggestion
 * @param {Boolean} [options.enableDebugWireframe] For debugging only. This must be true for debugWireframe to work in WebGL1. This cannot be set after the tileset has loaded.
```

File: Source/Scene/Cesium3DTileset.js [hunk 5]
```diff
@@ -1358,11 +1341,8 @@ Object.defineProperties(Cesium3DTileset.prototype, {
 
   /**
    * A custom shader to apply to all tiles in the tileset. Only used for
-   * contents that use {@link ModelExperimental}. Using custom shaders with a
+   * contents that use {@link Model}. Using custom shaders with a
    * {@link Cesium3DTileStyle} may lead to undefined behavior.
```
Comment (j9liu) [inline Source/Scene/Cesium3DTileset.js:1345]: Is there a type of content that doesn't use `Model`? If not, remove the "Only used..." sentence

File: Specs/Scene/ModelExperimental/ModelExperimentalSpec.js [hunk 6]
```diff
@@ -24,7 +24,7 @@ import {
   JulianDate,
   Math as CesiumMath,
   Matrix4,
-  ModelExperimental,
```
Comment (j9liu) [inline Specs/Scene/ModelExperimental/ModelExperimentalSpec.js:27]: This spec needs to be renamed

File: Specs/Scene/PostProcessStageLibrarySpec.js [hunk 7]
```diff
@@ -119,7 +119,8 @@ describe(
       });
     });
 
-    it("per-feature black and white", function () {
+    // glTF 1.0 model needs to be converted to 2.0
```
Comment (j9liu) [inline Specs/Scene/PostProcessStageLibrarySpec.js:122]: Can this be replaced with a glTF 2.0 box?

File: Specs/Scene/ModelExperimental/loadAndZoomToModelExperimental.js [hunk 8]
```diff
@@ -1,11 +1,11 @@
-import { ModelExperimental } from "../../../Source/Cesium.js";
+import { Model } from "../../../Source/Cesium.js";
```
Comment (j9liu) [inline Specs/Scene/ModelExperimental/loadAndZoomToModelExperimental.js:1]: This file and function should be renamed

File: Documentation/CustomShaderGuide/README.md [hunk 9]
```diff
@@ -76,7 +76,7 @@ const tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
 }));
 
 // Applying to a model directly
```
Comment (j9liu) [inline Documentation/CustomShaderGuide/README.md:78]: Remove the mention of `enableModelExperimental` above this too.

File: Source/Scene/Cesium3DTileset.js [hunk 10]
```diff
@@ -13,7 +13,6 @@ import destroyObject from "../Core/destroyObject.js";
 import DeveloperError from "../Core/DeveloperError.js";
 import Ellipsoid from "../Core/Ellipsoid.js";
 import Event from "../Core/Event.js";
-import ExperimentalFeatures from "../Core/ExperimentalFeatures.js";
```
Comment (j9liu) [inline Source/Scene/Cesium3DTileset.js:16]: Can you update the documentation for `this.showOutline` and `this.outlineColor`

**Review by j9liu — CHANGES_REQUESTED** [review]
@ptrgags I did a first sweep and left some comments. While you're looking through them, I'll go through all of the sandcastles and make sure they pass

File: Source/Scene/Cesium3DTileset.js (hunk 5 above)
Comment (ptrgags) [reply Source/Scene/Cesium3DTileset.js:1345]: vector and geometry contents don't use the new `Model` at present.

**Comment by j9liu** [comment]
@ptrgags Oh, can you also remove the `development/3D Models Instancing` sandcastle?

We should also consolidate the original `3D Models` sandcastle with the `Model Experimental 3D Models` one, at some point...

File: Specs/Scene/PostProcessStageLibrarySpec.js (hunk 7 above)
Comment (ptrgags) [reply Specs/Scene/PostProcessStageLibrarySpec.js:122]: Possibly, let me see if I can drop in `BoxTextured` (I don't think we have just a 2.0 box). If not, I'll update this when I work on cleaning up glTF 1.0 assets

Comment (ptrgags) [reply Specs/Scene/PostProcessStageLibrarySpec.js]: well, I got one more spec to work, but the others will take more thought, probably have to adjust the camera setup.

**Comment by ptrgags** [comment]
@j9liu updated!

**Comment by j9liu** [comment]
@ptrgags looks good, I just pushed a fix for a lingering typo. thanks!

**Pull request merged**